en/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md

```diff
@@ -0,0 +1,137 @@
+---
+id: applications-cisco-ssms-restapi
+title: Cisco SSMS
+---
+
+## Overview
+
+Cisco Smart Software Manager On-Prem (formerly known as Cisco Smart Software Manager satellite) is a component of Cisco Smart Licensing
+that works in conjunction with Cisco Smart Software Manager (SSM).
+It offers near real-time visibility and reporting of the Cisco licensesyou purchase and consume while giving security-sensitive organizations,
+a way to access a subset of Cisco SSM functionality without using a direct Internet connection to manage their install base.
+
+## Plugin Pack Assets
+
+### Monitored Objects
+
+* Applications
+* Servers
+* Licences
+* Alerts
+
+### Collected Metrics
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Alerts-->
+
+| Metric Name                | Description              |
+| :------------------------- | :------------------------|
+| account.alerts.minor.count | Counter of alerts minor. |
+| account.alerts.major.count | Counter of alerts major. |
+
+
+<!--Licenses-->
+
+| Metric Name                | Description                                        |
+| :------------------------- | :------------------------------------------------- |
+| licenses.usage.count       | Counter of licenses usage. Unit : Bytes (B)        |
+| licenses.free.count        | Counter of licenses free. Unit : Bytes (B)         |
+| licenses.usage.percentage  | Percentage of licenses usage. Unit : percentage(%) |
+
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+
+## Prerequisites
+
+### API Privileges
+
+A service account is required to query the Cisco SSMS API. It must have sufficient read privileges in the account targeted.

```

Comment:
Maybe add a reference to the documentation like this : https://www.cisco.com/c/dam/en_us/buy/smart-accounts/smart-software-manager-satellite-enhanced-edition-6-2-0-user-guide.pdf

Reply:
Added the reference.